packages/constants/src/lib/constants/constants.ts

```diff
@@ -1018,3 +999,194 @@ export const RELAY_URL_DATIL_TEST = 'https://datil-test-relayer.getlit.dev';
 // ========== Lit Actions ==========
 export const LIT_ACTION_IPFS_HASH =
   'QmUjX8MW6StQ7NKNdaS6g4RMkvN5hcgtKmEi8Mca6oX4t3';
+
+// ========== Chains ==========
+export const VMTYPE = {
+  EVM: 'EVM',
+  SVM: 'SVM',
+  CVM: 'CVM',
+} as const;
+export type VMTYPE_TYPE = keyof typeof VMTYPE;
+export type VMTYPE_VALUES = (typeof VMTYPE)[keyof typeof VMTYPE];
+
+export const LIT_CURVE = {
+  BLS: 'BLS',
+  EcdsaK256: 'K256',
+  EcdsaCaitSith: 'ECDSA_CAIT_SITH', // Legacy alias of K256
+  EcdsaCAITSITHP256: 'EcdsaCaitSithP256',
+} as const;
+export type LIT_CURVE_TYPE = keyof typeof LIT_CURVE;
+// This should replicate SigShare.sigType in types package
+export type LIT_CURVE_VALUES = (typeof LIT_CURVE)[keyof typeof LIT_CURVE];
+
+// ========== Either Types ==========
+export const EITHER_TYPE = {
+  ERROR: 'ERROR',
+  SUCCESS: 'SUCCESS',
+} as const;
+export type EITHER_TYPE_TYPE = keyof typeof EITHER_TYPE;
+export type EITHER_TYPE_VALUES = (typeof EITHER_TYPE)[keyof typeof EITHER_TYPE];
+
+// ========== Supported PKP Auth Method Types ==========
+export const AuthMethodType = {
+  EthWallet: 1,
+  LitAction: 2,
+  WebAuthn: 3,
+  Discord: 4,
+  Google: 5,
+  GoogleJwt: 6,
+  AppleJwt: 8,
+  StytchOtp: 9,
+  StytchEmailFactorOtp: 10,
+  StytchSmsFactorOtp: 11,
+  StytchWhatsAppFactorOtp: 12,
+  StytchTotpFactorOtp: 13,
+} as const;
+export type AuthMethodType_TYPE = keyof typeof AuthMethodType;
```

Comment:
Is this because we want to maintain the original name in camel case `AuthMethodType` and adding our new naming convention `_TYPE` and `_VALUES` like `LIT_NETWORK_TYPES`?

Reply:
Let's use this chance to unify all constants naming conventions and expose the previous names as deprecated values so we end up with proper names but breaking as little as possible